### PR TITLE
Api url remove

### PR DESCRIPTION
**Pull request by kratigupta99**

### CODE REVIEW

**Review by kjrach — DISMISSED** [review]
Looks good. Thanks.
Next steps:
- Once PR is reviewed and merged, cut a release in this repo. 
- The nightly job in https://github.com/HPE/terraform-provider-hpegl will raise a PR for the latest release cut.
-  Inform the team to update docs accordingly if not updated as a part of the PR raised.
-  Once the PR in https://github.com/HPE/terraform-provider-hpegl is merged, and a release is cut by them in this repo, verify if the terraform registry has the changes updated (https://registry.terraform.io/providers/HPE/hpegl/latest/docs/resources/caas_cluster)

File: README.md [hunk 1]
```diff
@@ -252,6 +252,21 @@ Note: Ensure there is no .terraform.lock.hcl or .terraform in examples/cluster-c
 
 Update examples/resources/hpegl_caas_cluster/resource.tf with all the the necessary values
 
+For dev/integ setup , the provider to be used in the .tf file is :
```
Comment (akashpatel2) [inline README.md]: i.e. setup,

need to remove extra space

File: README.md [hunk 2]
```diff
@@ -252,6 +252,21 @@ Note: Ensure there is no .terraform.lock.hcl or .terraform in examples/cluster-c
...
+```bash
+provider hpegl {
+  caas {
+    api_url = "https://mcaas.intg.hpedevops.net/mcaas"
+  }
+}
+```
+For prod setup , the provider to be used in the .tf file is :
```
Comment (akashpatel2) [inline README.md]: same here

**Review by akashpatel2 — DISMISSED** [review]
LGTM, minor comments
Need to make sure;
1) after PR merge cut a release on master branch
2) release artifact in parent repo
3) verify public doc

**Comment by kratigupta99** [comment]
> LGTM, minor comments Need to make sure;
> 
> 1. after PR merge cut a release on master branch
> 2. release artifact in parent repo
> 3. verify public doc

Sure, will do that.

**Review by akashpatel2 — APPROVED** [review]

**Pull request merged**